claudio-plugin/skills/google-drive-reader/scripts/search_files.sh

```
@@ -0,0 +1,142 @@
+    local parts=()
+    parts+=("trashed = false")
+    # Search name and full text
+    parts+=("(name contains '${QUERY_TERM}' or fullText contains '${QUERY_TERM}')")
+
+    if [ -n "$TYPE_FILTER" ]; then
+        local mime
+        mime=$(resolve_mime_type "$TYPE_FILTER")
+        parts+=("mimeType = '${mime}'")
+    fi
+
+    local IFS=" and "
```

Shellcheck (0.11.0)

[warning] This IFS value contains the literal letter 'n'. For tabs/linefeeds/escapes, use $'..', literal, or printf.

(SC2141)